flocker/node/backends.py

```diff
@@ -90,6 +90,23 @@ class BackendDescription(PClass):
     :ivar deployer_type: A constant from ``DeployerType`` indicating which kind
         of ``IDeployer`` the API object returned by ``api_factory`` is usable
         with.
+
+    The following fields are experimental, and subject to change
+    without notice.
+
+    :ivar acceptance_provision_node: Function that will provision a
```

Comment:
Should this really be "acceptance"? Rather than naming it after how we're using it now, could we find a name that's more intrinsic to what it is?